rackcity/utils/change_planner_utils.py

```diff
@@ -178,3 +178,117 @@ def get_page_count_response_for_cp(request, change_plan):
     count = assets.count() + assetsCP.count()
     page_count = math.ceil(count / page_size)
     return JsonResponse({"page_count": page_count})
+
+
+def get_changes_on_asset(asset, assetCP):
+    fields = [field.name for field in Asset._meta.fields]
+    changes = []
+    for field in fields:
+        if getattr(asset, field) != getattr(assetCP, field):
+            # TODO will need to serialize field if it's another model
```

Comment:
so this would be rack/model... what about network and power connections? since those aren't fields on asset. You could you reverse relations to get all the network connections and power connections that Asset and AssetCP are forieign keys for and compare those? if you want to leave that in a separate PR im also good with that.

Reply:
but we would need to do that in order to highlight changes to network /power connections